<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="loginEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Login...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordEdit">
       <property name="placeholderText">
        <string>Password...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="signInBtn">
       <property name="text">
        <string>Sign in</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
